nixos/modules/services/misc/radicle.nix

```diff
@@ -0,0 +1,515 @@
+{ config, lib, pkgs, options, ...  }:
```

Comment:
```suggestion
{ config, lib, pkgs, ...  }:
```

Reply:
`options` is actually used to fetch the type of an nginx virtualHost.

It's like what's done in some modules in `nixos/modules/services/web-apps/`:
```
type = with types; nullOr (submodule (import ../web-servers/nginx/vhost-options.nix { inherit config lib; }));
```
but without re-importing any file.